### PR TITLE
use the projects cards api end points for signed in home page

### PR DESCRIPTION
**Pull request by camallen**
Use the project cards route for the signed in home page for more efficient api resource usage that should speed up the render times (less network time).

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [ ] Did you deploy a staging branch?

## Optional

- [ ] If it's a new component, is it in ES6? Is it clear of warnings from ESLint?
- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?

### CODE REVIEW

File: app/pages/home-for-user/my-builds.jsx [hunk 1]
```diff
@@ -101,8 +88,7 @@ const MyBuildsSection = React.createClass({
 
         <div className="project-card-list">
           {this.state.projects.map((project) => {
-            const avatarSrc = !!this.state.avatars[project.id] ? this.state.avatars[project.id].src : null;
-            return <ProjectCard key={project.id} project={project} imageSrc={avatarSrc} href={`/lab/${project.id}`} />;
+            return <ProjectCard key={project.id} project={project} href={`/lab/${project.id}`} />;
```
Comment (camallen) [inline app/pages/home-for-user/my-builds.jsx]: @srallen passing the project prop in here has a cut down resource response, e.g. no project.tags attributes. so when following the link the lab page falls over as it crashes trying to split the tags

**Comment by camallen** [comment]
I'm having troubles getting this sorted, the `resource.refresh(true)` function works but doesn't seem to emit the changes correctly to the resource (it prints fine but the attribute methods don't return the refreshed state). If someone is available to pair on this next week, i'll have a crack on it then.

**Comment by camallen** [comment]
i ended up just using the cards params for the recent projects end points and left my builds as full resource requests with included avatars. It should speed up the 'my recent projects' ui component and not do much on the 'my builds'

**Comment by wgranger** [comment]
This looks to be more efficient. Good stuff.

**Pull request merged**